ext/sqlite3/sqlite3.c

```diff
@@ -1682,9 +1680,7 @@ static int register_bound_parameter_to_sqlite(struct php_sqlite3_bound_param *pa
 		return 0;
 	}
 
-	if (param->param_number >= 1) {
```

Comment:
If you remove here, what happens just above ?

Reply:
I don't understand the question. We early exit when `param->param_number < 1`, therefore we know that `param->param_number >= 1` must be true at this point.

Reply:
Ah ok I see yes I asked if the previous condition could happen.